manifests/manifests-maistra/maistra.package.yaml

```diff
@@ -0,0 +1,8 @@
+packageName: maistraoperator
+channels:
+- name: "1.0"
+  currentCSV: maistraoperator.v1.1.0
+channels:
```

Comment:
this looks wrong. my two `channels` keys here?

Reply:
We're moving from `1.0` to `stable`.  The `1.0` channel is provided for existing users.  New users should use the `stable` channel.  Eventually, the `1.0` channel will be removed.

Reply:
+1, but there shouldn't be two `channels:` entries in the root, right?

Reply:
Doh! Thanks @dgn.  pushed the fix.